common/api-review/firestore-exp.api.md

```diff
@@ -278,8 +278,33 @@ export function limit(limit: number): QueryConstraint;
 // @public
 export function limitToLast(limit: number): QueryConstraint;
 
+// @public
+export function loadBundle(firestore: FirebaseFirestore, bundleData: ReadableStream<Uint8Array> | ArrayBuffer | string): LoadBundleTask;
+
+// @public
+export class LoadBundleTask implements PromiseLike<LoadBundleTaskProgress> {
+    catch<R>(onRejected: (a: Error) => R | PromiseLike<R>): Promise<R | LoadBundleTaskProgress>;
+    _completeWith(progress: LoadBundleTaskProgress): void;

```

Comment:
I think these _ methods should be tagged `@internal` in packages/firestore/src/exp/bundle.ts if we don't want to expose them publicly?

Reply:
I think @schmidt-sebastian's script strips things with a leading underscore automatically in the d.ts file, but not in the api report.

Ideally this file should be consistent with d.ts. Maybe we can extend the script to update the api report too?

Reply:
We don't have to do it in this PR of course.

Reply:
Waiting for review :) https://github.com/firebase/firebase-js-sdk/pull/4207